llm/ggml.go

```diff
@@ -43,7 +46,7 @@ type FileType interface {
 	String() string
 }
 
-type GGML struct {
+type ModelFile struct {
```

Comment:
I don't think this rename is necessary. The high level concept is still GGML so the name should continue